### PR TITLE
Gslux 705 modify feature selection and sync between map and UI

### PR DESCRIPTION
**Pull request by mki-c2c**
start of work for selecting and editing features

### CODE REVIEW

**Comment by mki-c2c** [comment]
all tests are fixed, would be good to merge this also.

File: src/components/draw/draw-panel-features.vue [hunk 1]
```diff
@@ -9,35 +10,39 @@ import { DrawnFeature } from '@/services/draw/drawn-feature'
...
+ *     if (oldFeatures === undefined || newFeatures.length > oldFeatures.length) {
+ *       const currentFeature =
+ *         newFeatures[oldFeatures === undefined ? 0 : newFeatures.length - 1]
+ *       // currentOpenedFeature.value = currentEditingFeature.value =
+ *       currentFeature.id
+ *     }
+ *   },
+ *   { immediate: true }
+ * )
+ * */
```
Comment (tkohr) [inline src/components/draw/draw-panel-features.vue:45]: I guess we can get rid of this here then.

File: src/services/state-persistor/state-persistor-features.mapper.ts [hunk 2]
```diff
@@ -1,14 +1,6 @@
...
 import { DrawnFeature } from '@/services/draw/drawn-feature'
 
-// const drawnFeatureToFeature = function (
-//   drawnFeature: DrawnFeature
-// ): Feature<Geometry> {
-//   const olFeature = drawnFeature.olFeature
-//   olFeature.set('name', drawnFeature.label)
-//   return olFeature
-// }
-
```
Comment (tkohr) [inline src/services/state-persistor/state-persistor-features.mapper.ts:11]: Can we also delete the commented lines below in `featuresToUrl`?

File: src/services/state-persistor/state-persistor-features.service.ts [hunk 3]
```diff
@@ -35,6 +36,8 @@ class StatePersistorFeaturesService implements StatePersistorService {
 
   restore() {
     const { drawnFeatures } = storeToRefs(useDrawStore())
+    // initialise map listeners for feature selection
+    useDrawSelect()
```
Comment (tkohr) [inline src/services/state-persistor/state-persistor-features.service.ts:40]: It feels a little strange to call a composable in a state persistor, but I agree that it makes sense to concentrate this logic in a composable and don't have a better suggestion neither.

**Review by tkohr — APPROVED** [review]
Thanks, @mki-c2c ! I just noticed that the label is not displayed bigger on selection yet, but we can handle this in a follow-up PR, just as adding e2e tests for the selecting.

**Comment by mki-c2c** [comment]
> Thanks, @mki-c2c ! I just noticed that the label is not displayed bigger on selection yet, but we can handle this in a follow-up PR, just as adding e2e tests for the selecting.
> 
> Not sure if you should rather merge #135 first, though, since the CI has not run on this one yet.

Yes, I would also like to see the other PR merged first.

On my local server, the selected labels appear bigger. We can check it with the CD version before merging into main

**Pull request merged**